### ISSUE TITLE
Dependency egm96 is not compatible with browser

### ISSUE THREAD

**Issue by sarfata** [JuanIrache/gopro-telemetry#115]
The [package `egm96`](https://github.com/jleppert/egm96/blob/master/egm96.js) used to correct the GPS altitudes has the following line in it:

```javascript
var data = fs.readFileSync(path.join(__dirname,'/WW15MGH.DAC'));
```

This does not work in the browser because `fs.readFileSync` is only available in Node.

I can see a few solutions:

 - Remove this dependency since it seems it is only required for old cameras (but that would mean getting wrong data for old cameras and is obviously a regression)
 - Change `egm96` to only load the data the first time it is used. This way if you disable GPS correction in the browser you can use the rest of the library. Altitude correction will not work in the browser.
 - Pass the correction data as an argument to the `egm96` package so the caller is responsible for loading it (via `readFileSync` on node - or via `fetch` in the browser)

I can make a pull request to implement one of these solutions but it seems you have already implemented a solution for https://goprotelemetryextractor.com/. Can you share what solutions you have chosen?

**Comment by JuanIrache** [comment]
Hi @sarfata , My online tool works with this code because gopro-telemetry is run server-side. I think the ideal solution would be something close to your third option, basically storing the data in a format both the browser and NodeJS can use.

We could also rely on an external API to provide the altitude correction data. Something like this: https://geodesy.noaa.gov/web_services/geoid.shtml

The problem with the API approach is it won't work when offline, though, which is important in some implementations.

Happy to hear your thoughts. Thanks for contributing.

**Comment by JuanIrache** [comment]
I believe this should be fixed in afbce18e2a104361b068d23f5d32f28c9dc2204f

Could you confirm it's the case in your implementation? It was never an issue on mine.

Thanks
